README.md

```diff
@@ -139,12 +139,14 @@ The full CLI command reference is available [on the web](https://www.balena.io/d
 
 ## Support, FAQ and troubleshooting
 
-If you come across any problems or would like to get in touch:
+To learn more, troubleshoot issues, or to contact us for support:
 
-* Check our [FAQ / troubleshooting document](https://github.com/balena-io/balena-cli/blob/master/TROUBLESHOOTING.md).
-* Ask us a question through the [balenaCloud forum](https://forums.balena.io/c/balena-cloud).
-* For bug reports or feature requests,
-  [have a look at the GitHub issues or create a new one](https://github.com/balena-io/balena-cli/issues/).
+* Check the [masterclass tutorials](https://www.balena.io/docs/learn/more/masterclasses/overview/)
+* Check our [FAQ / troubleshooting document](https://github.com/balena-io/balena-cli/blob/master/TROUBLESHOOTING.md)
+* Ask us a question through the [balenaCloud forum](https://forums.balena.io/c/balena-cloud)
+
+For CLI bug reports or feature requests, check the
+[CLI GitHub issues](https://github.com/balena-io/balena-cli/issues/).
```

Comment:
These changes are reflected in `cli.markdown`, and thus eventually also in the online "reference" docs web page:
https://www.balena.io/docs/reference/balena-cli/#support-faq-and-troubleshooting

My main intention in this commit was to include references to the masterclasses and FAQ document in CLI help/error output, in addition to the references to the forums and GitHub issues. My first attempt was to literally add the URLs for the masterclasses and FAQ to `messages.ts` (`reachingOut` and `help` variables), but then I found that the output was too verbose (too many printed lines, distracting). Then I thought of replacing those 4 URLs (masterclasses, FAQ, forums and GitHub issues) with a single URLs to the reference docs website, where the other URLs are listed.

Reply:
Yeah this is a nice improvement.